CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Additions and Improvements
 - Explain and improve price validation for London and local transactions during block proposal selection [#4602](https://github.com/hyperledger/besu/pull/4602)
 - Support for ephemeral testnet Shandong.  EIPs are still in flux, besu does not fully sync yet, and the network is subject to restarts. [#//FIXME](https://github.com/hyperledger/besu/pull///FIXME)
+- Added `--rpc-max-logs-range` CLI option to allow limiting the number of blocks queried by `eth_getLogs` RPC API [#4597](https://github.com/hyperledger/besu/pull/4597)
```

Comment:
think we need to add a note that this is a breaking change - if you have a workflow that depends on requesting eth_getLogs for a block range that is larger than the new default X

Reply:
See previous comment - a default hasn't been added _yet_.